pkgs/development/python-modules/pyqtdatavisualization/default.nix

```
@@ -0,0 +1,76 @@
+{ lib
+, stdenv
+, buildPythonPackage
+, fetchPypi
+, pyqt5
+, pyqt-builder
+, python
+, pythonOlder
+, qtdatavis3d
+, setuptools
+, sip
+}:
+
+buildPythonPackage rec {
+  pname = "PyQtDataVisualization";
+  version = "5.15.5";
+  format = "pyproject";
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchPypi {
+    inherit pname version;
```

```suggestion
  pname = "pyqtdatavisualization";
  version = "5.15.5";
  format = "pyproject";

  disabled = pythonOlder "3.7";

  src = fetchPypi {
    pname = "PyQtDataVisualization";
    inherit version;
```
we normalize the pnames and attrs, see the python docs

---

Thanks for the hint. I should have read the python chapter in the manual before packaging python modules.